rtl/eclrtl/eclhelper_dyn.cpp

```diff
@@ -136,52 +136,7 @@ class ECLRTL_API CDynamicDiskReadArg : public CThorDiskReadArg
             MemoryBuffer lobuffer;
             MemoryBuffer hibuffer;
             Owned<IStringSet> filterSet = createStringSet(fieldSize);
```

Comment:
Should switch this to use the new ValueSet I would think (maybe can't for IndexRead case, but no reason not to for diskread is there?)

Reply:
I thought I would leave that for a separate PR